source/git-api.js

```diff
@@ -169,8 +169,9 @@ exports.registerApi = (env) => {
   }
 
   const credentialsOption = (socketId, remote) => {
+    const rootPath = _.trim(config.rootPath || '', '/'); // remove '/', because of windows git issue
     const credentialsHelperPath = path.resolve(__dirname, '..', 'bin', 'credentials-helper').replace(/\\/g, '/');
-    return ['-c', `credential.helper=${credentialsHelperPath} ${socketId} ${config.port} ${remote}`];
+    return ['-c', `credential.helper=${credentialsHelperPath} ${socketId} ${config.port} ${remote} ${rootPath}`];
```

Comment:
Instead of passing in `rootPath` as optional argument, can we potentially do something like this below and change the name of 3rd argument name in the `credential-helper.js` and it will introduce minimal changes?

```js
let portAndRootPath = `${config.port}`
if (rootPath) {
  portAndRootPath = `${config.port}/${rootPath}`
}
credential.helper=${credentialsHelperPath} ${socketId} ${portAndRootPath} ${remote}
```